tests/fullbench.c

```diff
@@ -268,6 +271,13 @@ static size_t benchMem(const void* src, size_t srcSize, U32 benchNb)
         if (g_zcc==NULL) g_zcc = ZSTD_createCCtx();
         break;
     case 12 :
+        if (g_zcc==NULL) g_zcc = ZSTD_createCCtx();
+        if (g_zcc) {
+            seqStore_t* seqStorePtr = ZSTD_getSeqStore(g_zcc);
+            if (seqStorePtr) seqStorePtr->forceExtDict = 1;
```

Comment:
It's a source of future maintenance difficulties (though hopefully its impact is likely going to be limited to `fullbench`). Nothing dramatic though, it's part of the trade-off to find.